docs/resources/Guild.md

```diff
@@ -371,6 +372,18 @@ A partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object. Represents an Offl
 }
 ```
 
+###### Guild Member Flags
+
+| Flag                                    | Value  | Description                                                   | Allows Modification |
+| ----------------------------------------| ------ | ------------------------------------------------------------- |---------------------|
+| DID_REJOIN                              | 1 << 0 | Member has leave and rejoined the guild                       | 0                   |
+| COMPLETED_ONBOARDING                    | 1 << 1 | Member has completed onboarding                               | 0                   |
+| BYPASSES_VERIFICATION                   | 1 << 2 | Member bypasses guild verification requirements               | 1                   |
+| STARTED_ONBOARDING                      | 1 << 3 | Member has started onboarding                                 | 0                   |
```

Comment:
Are these 0 and 1 supposed to be booleans? Usually the docs use "false" and "true" which is also clearer to the reader. See [Message Type Docs](https://discord.com/developers/docs/resources/channel#message-object-message-types)